src/Simulation/Simulators.Tests/Tests.Microsoft.Quantum.Simulation.Simulators.csproj

```diff
@@ -38,12 +41,17 @@
   <Target Name="PostBuild" AfterTargets="PostBuildEvent">
     <PropertyGroup>
       <_ExeDir>$(MSBuildThisFileDirectory)TestProjects\QsharpExe\built\</_ExeDir>
+      <_ClassicControlDir>$(MSBuildThisFileDirectory)TestProjects\ClassicControlSupportTest\built\</_ClassicControlDir>
+      <ExecutionTarget>$(MSBuildThisFileDirectory)..\TestTargets\Simulation\Example\bin\$(Configuration)\netcoreapp3.1\Example.dll</ExecutionTarget>
```

Comment:
What were you planning to use that for?

Reply:
I copied what was done for the QsharpExe project. Should I have something else here?

Reply:
Looking at the diff, the ExecutionTarget property seems to have been added and is not used by anything.

Reply:
I've removed the ExecutionTarget tag.